lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_download_translations_action.rb

```diff
@@ -48,7 +48,7 @@ def self.available_options
             env_name: 'FL_DOWNLOAD_TRANSLATIONS_RES_DIR',
             description: "The path to the Android project's `res` dir (typically the `<project name>/src/main/res` directory) containing the `values-*` subdirs",
             type: String,
-            default_value: "#{ENV['PROJECT_NAME']}/src/main/res"
+            default_value: "#{ENV.fetch('PROJECT_NAME', nil)}/src/main/res"
```

Comment:
From what I've read, `ENV.fetch` will produce an error if the environment variable isn't found, whereas `ENV[]` will just pass in `nil` and not produce an error. The default of `nil` was automatically added by Rubocop, but does that negate some of the benefit of using `ENV.fetch` if it still defaults to `nil` when the variable isn't found? 

Rubocop doesn't say a default is needed when using `ENV.fetch`, but I left it in there for each of the `ENV.fetch` corrections so I could ask about it here.

Reply:
Yes, I believe that `ENV.fetch('PROJECT_NAME')` would crash is the env var was not set, while `ENV.fetch('PROJECT_NAME', nil)` would return nil if the env var was not set… just like `ENV['PROJECT_NAME']` would. So in this particular condition, I'm not sure I see the benefit of `ENV.fetch('PROJECT_NAME', nil)` over `ENV['PROJECT_NAME']`.

If anything, what would probably make sense if `PROJECT_NAME` was not set, would be to use the `./` current directory as a fallback instead. So I'd instead maybe suggest `ENV.fetch('PROJECT_NAME', '.')` instead?

Reply:
I decided to keep these as `ENV.fetch`'s instead of reverting to `ENV[]` or setting defaults (at least for now). My thinking is that if one of these environment variables isn't set, we wouldn't necessarily want a default value to be swapped in. That seems like it could lead to unexpected changes when running the toolkit actions. I would say that maybe we should remove the default and have the commands crash if a variable isn't set, but I think that's a discussion worth having outside this PR.

Reply:
@AliSoftware what do you think about that reasoning?

Reply:
Problem is the default value is evaluated even if you provide one when the action is parsed by fastlane. Simply asking fastlane for the documentation of the action (`fastlane action name_of_an_action`) would crash if one of the env var was not set.

Reply:
@AliSoftware But not if we leave the defaults set as `nil`, right? It should act how `ENV[]` currently acts and just do nothing. 

When I ran these lines: 
```
lane :test_env do    
    puts ENV['EMPTY_CONSTANT_1']
    puts ENV.fetch('EMPTY_CONSTANT_2', nil)
    puts ENV.fetch('EMPTY_CONSTANT_3')
  end
```

It didn't error until hitting the `EMPTY_CONSTANT_3` line. So if we do have an env var missing but the default value is set as `nil`, it should perform exactly like `ENV[]` works now if an env var is missing.

Reply:
Sorry if I'm going around in circles 😄

Reply:
> `ENV.fetch(key)` without a fallback value is not something we should use inside a `default_value` of a `ConfigItem`

Ah okay, that makes sense. Thank you for helping me understand! 

I rolled back the `ENV.fetch` changes to `ENV[]` and added a Rubocop exception. Hopefully this will all be moot soon anyway as we work towards not using as many env vars in these actions 🙂